src/app.py

```diff
@@ -936,37 +939,45 @@ def submit_dataset(uuid):
-            dataset_request[HubmapConst.DATASET_RUN_ID] = submission_data['run_id']
-        else:
-            logger.error('Failed call to AirFlow HTTP Response: ' + str(r.status_code) + ' msg: ' + str(r.text)) 
-            return Response("Ingest pipeline failed: " + str(r.text), r.status_code)
-        
-        dataset_request['status'] = 'Processing'
-        put_url = commons_file_helper.ensureTrailingSlashURL(app.config['ENTITY_WEBSERVICE_URL']) + 'entities/' + uuid
-        response = requests.put(put_url, json = dataset_request, headers = {'Authorization': 'Bearer ' + token, 'X-Hubmap-Application':'ingest-api' }, verify = False)
-        if not response.status_code == 200:
-            logger.error(f"call to {put_url} failed with code:{response.status_code} message:" + response.text)
-            return Response(response.text, response.status_code)
-        updated_dataset = response.json()
-        return jsonify(updated_dataset)
-    except HTTPException as hte:
-        return Response(hte.get_description(), hte.get_status_code())
     except Exception as e:
         logger.error(e, exc_info=True)
-        return Response("Unexpected error while creating a dataset: " + str(e) + "  Check the logs", 500)        
- 
+        return Response("Unexpected error while creating a dataset: " + str(e) + "  Check the logs", 500)
+    dataset_request['status'] = 'Processing'
+    put_url = commons_file_helper.ensureTrailingSlashURL(app.config['ENTITY_WEBSERVICE_URL']) + 'entities/' + uuid
+    response = requests.put(put_url, json=dataset_request,
+                            headers={'Authorization': 'Bearer ' + token, 'X-Hubmap-Application': 'ingest-api'},
+                            verify=False)
+    if not response.status_code == 200:
+        error_msg = f"call to {put_url} failed with code:{response.status_code} message:" + response.text
+        logger.error(error_msg)
+        return Response(str(error_msg), 500)
+    def call_airflow():
+        try:
+            r = requests.post(pipeline_url, json={"submission_id" : "{uuid}".format(uuid=uuid), "process" : app.config['INGEST_PIPELINE_DEFAULT_PROCESS'],"full_path": ingest_helper.get_dataset_directory_absolute_path(dataset_request, group_uuid, uuid),"provider": "{group_name}".format(group_name=AuthHelper.getGroupDisplayName(group_uuid))}, headers={'Content-Type':'application/json', 'Authorization': 'Bearer {token}'.format(token=AuthHelper.instance().getProcessSecret() )}, verify=False)
+            if r.ok == True:
+                """expect data like this:
+                {"ingest_id": "abc123", "run_id": "run_657-xyz", "overall_file_count": "99", "top_folder_contents": "["IMS", "processed_microscopy","raw_microscopy","VAN0001-RK-1-spatial_meta.txt"]"}
+                """
+                data = json.loads(r.content.decode())
+                submission_data = data['response']
+                dataset_request[HubmapConst.DATASET_INGEST_ID_ATTRIBUTE] = submission_data['ingest_id']
```

Comment:
We had plan to deprecate the constants defined in hubmap commons, so it's a good time to do it gradually by just hardcoding the fields:

```
 DATASET_INGEST_ID_ATTRIBUTE = 'ingest_id'
 DATASET_RUN_ID = 'run_id'
 ```

Reply:
Do you want those constants defined in app.cfg or just at the top of this endpoint.

Reply:
@DerekFurstPitt we don't have to use any constants for now, just hardcode them as
```
dataset_request['ingest_id'] = submission_data['ingest_id']
dataset_request['run_id'] = submission_data['run_id']
```